src/main/java/dev/Roach/AlertService.java

```diff
@@ -0,0 +1,85 @@
+package dev.Roach;
+
+import dev.Roach.datamodel.game.Game;
+import dev.Roach.datamodel.game.GamePojo;
+import dev.Roach.fetchers.GamesFetcher;
+import lombok.Setter;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.List;
+
+@Setter
+public class AlertService {
+
+    private HttpClient httpClient = HttpClient.newBuilder().build();
+
+    private GamesFetcher gamesFetcher = new GamesFetcher();
+
+    public boolean createOrUpdateAlert(String email, int gameID, double price) {
+        String url = String.format("https://www.cheapshark.com/api/1.0/alerts?action=set&email=%s&gameID=%d&price=%.2f",
+                email, gameID, price);
+        return sendAlertRequest(url);
+    }
+
+    public boolean deleteAlert(String email, int gameID) {
+        String url = String.format("https://www.cheapshark.com/api/1.0/alerts?action=delete&email=%s&gameID=%d",
+                email, gameID);
+        return sendAlertRequest(url);
+    }
+
+    private boolean sendAlertRequest(String url) {
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create(url))
+                .build();
+        try {
+            HttpResponse<String> response = httpClient.send(request, HttpResponse.BodyHandlers.ofString());
+            return Boolean.parseBoolean(response.body());
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+
+        }
+    }
+
+    public String getAlertsForEmail(String email) {
+
+        String url = String.format("https://www.cheapshark.com/api/1.0/alerts?action=manage&email=%s", email);
```

Comment:
Everything up to action= can be put into a variable to reduce the number of magic strings